chapters/Chapter 1 — Untitled.md

```diff
@@ -49,14 +49,76 @@ Something wet was pooling at the bottom of her helmet’s goggles. Right: she wa
 
 With the gravity field she should be good to run, but she kept her pace steady as she moved through her bedroom to the door of the hold, carefully ducking under the flying crate and batteries.
 
-Her hand reached for her gun as she opened the door, but there was no need. Her attacker was tumbling. His neck wasn’t broken yet: he was still managing to bring his arms up in time to protect himself, but he was tiring embarrassingly quickly. A few of his many weapons had detached themselves from him and were now flying around the hold with him.
+Her hand reached for her gun as she opened the door, but there was no need. Her attacker was tumbling. His neck wasn’t broken yet: he was still managing to bring his arms long up in time to protect himself, but he was tiring embarrassingly quickly. A few of his many weapons had detached themselves from him and were now flying around the hold with him.
 
 The hold door resealed itself behind Leanne. She must’ve flicked the switch out of habit—it was a good habit when on a spaceship.
 
 She’d closed it just in time. The forces of the spinning ships were far too much for their docking connectors to handle, and they wrenched themselves apart.
 
 Her attacker had left the airlocks open in case he had needed a quick escape. His escape was rather quicker than he intended: the air from the hold forced its way out, shoving him into space.
 
-Re: assassin flying out of airlock, and her after him:
+Leanne was not far behind: her gravity generator tried to keep her on the floor; her fingers brushed the griprail; it was not enough. 
 
-Later, she’d tell herself she intended it this way all along. But she more preferred the simpler, direct approach. Long-term gambles rarely paid.
+Later, she’d tell herself she intended it this way all along. She’d put the suit on, after all.
```

Comment:
👍 

She tells herself things a lot. She also tells herself quite often that she intended something, or must have intended it, etc. There's something else going on (obviously, since I'm calling attention to it in this comment), but I'm trying not to call _too_ much attention to it in the actual story. I want it to just grow slowly bit by bit as the story unfolds, until it's all revealed at the end.